services/web/server/src/simcore_service_webserver/users/_notifications.py

```diff
@@ -30,6 +31,7 @@ class BaseUserNotification(BaseModel):
     title: str
     text: str
     date: datetime
+    product: ProductName | Literal["UNDEFINED"]
```

Comment:
The order here matters since pydantic parses the unions in order and resolves the first one that validates. For that reason you should always start with the most strict constraint. Therefore this is more correct:
```python
product:  Literal["UNDEFINED"] | ProductName
```
For the moment, it does not have practical consequences since `ProductName` is basically a str but it might have in the near future if this definition is changed

Reply:
Thanks